boards/common/remote/include/periph_common.h

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
This file was removed recently. You had a rebase issue with this one. The `#ifdef MODULE_PERIPH_UART_HW_FC` should go here:
https://github.com/RIOT-OS/RIOT/blob/82b96ff831dff1ad1849eeecdf92b88a0c39fbd3/boards/common/remote/include/cfg_uart_default.h#L35-L51